CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# dbt_fivetran_log version.version
+# dbt_fivetran_log v1.11.0
+[PR #141](https://github.com/fivetran/dbt_fivetran_log/pull/141) includes the following updates:
 
-## Documentation
+## Schema Changes: Adding the Transformation Runs Table
+- We have added the `transformation_runs` source table. Note that not all customers have the `transformation_runs` source table, particularly if they are not using Fivetran Transformations. Therefore, the `transformation_runs` table will only populate if the table exists in your schema, via a new variable `fivetran_platform_using_transformations`, which automatically checks for the table. If the table doesn't exist, the staging `stg_fivetran_platform__transformation_runs` model will persist as an empty model and respective downstream fields will be null. 
+
+- If the `transformation_runs` source table exists in your schema, `fivetran_platform_using_transformations` will be set to True and the following updates apply:
+  - Added a new staging [`stg_fivetran_platform__transformation_runs`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.stg_fivetran_platform__transformation_runs) model. 
+    - We have also added the `get_transformation_runs_columns()` macro to ensure all required columns are present.
```

Comment:
Let's move this to an under the hood change since this isn't really applicable for users as a breaking schema change.

Reply:
updated some of the language in the CHANGELOG